functions/src/helpers.ts

```diff
@@ -16,23 +16,63 @@ export const createMP3DBRef = (filePath, _module) => {
     const mp3 = parts[2];
     let file_name = mp3.slice(0, -4);
 
-    return `/${_module}/files/${list}/${file_name}`;
+    return `/files/${list}/${file_name}`;
 }
 
 export const extractListFromFilename = (fileName) => {
     return fileName.match(/^[^-]*[^ -]/g)[0];
 }
 
 
-export const storeFileNameToDB = async (filePath, db, _module) => {
+export const storeFileToDB = async (filePath, db, _module) => {
     if(checkValidMP3(filePath)) {
         let ref = db.ref(createMP3DBRef(filePath, _module));
 
         //checking if the file already exists in the RT db
         let snapshot = await ref.child("status").once('value')
 
         if(!snapshot.exists()) {
-            ref.set({status: "Spare"});
+            ref.set({
+                languages: ['test'],
+                languagesConfirmed: false,
+                notes: 'test',
+                soundQualityReporting: {
+                    status: 'Spare',
+                    followUp: 'test',
+                    timestampDone: 'test',
+                    timestampGiven: 'test',
+                    asignee: {
+                        name: 'test',
+                        emailAddress: 'test'
+                    }
+                },
+                contentReporting: {
+                    status: 'Spare',
+                    followUp: 'test',
+                    timestampDone: 'test',
+                    timestampGiven: 'test',
+                    asignee: {
+                        name: 'test',
+                        emailAddress: 'test'
+                    },
+                    soundQualitySubmission: {
```

Comment:
Where have you taken this object path from? It is not in the data structure.

Reply:
![image](https://user-images.githubusercontent.com/1976151/48629418-3bbd7780-e9c2-11e8-9a4f-15f2dd7d0f0a.png)

Reply:
sorry

Reply:
Sorry I used the fields of **Sound Quality Reporting** instead of **Content Reporting**

In **Content Reporting**, it says at the end 
> submission, see **“Content Reporting Submissions”** below
![image](https://user-images.githubusercontent.com/1976151/48630039-d7031c80-e9c3-11e8-978f-22c9d1b5fd83.png)

where can I find the data structure for  **“Content Reporting Submissions”** ?

Reply:
You don't need to feel this content reporting submission while adding new file to the database. The submission is not yet done, so nothing to fill.